app/views/doctors/transcripts/show.haml

```diff
@@ -0,0 +1,13 @@
+%p
+  %b Description:
+  = @transcript.chat_room.urgent_request.description     
+%p
+  %b Patient:
+  = @transcript.chat_room.patient.name 
+%p
+  %b Transcript creation time:
+  = @transcript.updated_at
```

Comment:
Try `@transcript.updated_at.strftime('%d/%m/%y: %H:%M %p')` and have a look at the difference... 😉